modules/experimental-layers/src/bezier-curve-layer/bezier-curve-layer.js

```diff
@@ -32,10 +32,10 @@ const DEFAULT_COLOR = [0, 0, 0, 255];
 const defaultProps = {
   strokeWidth: 1,
```

Comment:
Prop type?

Reply:
Nit: Type `number` should be auto-inferred if the value is a number, at least by the 6.0 prop type system, so that explicit prop type would only be needed to set things like `min` and `max`. 

I admit that I am not up-to-date with the latest changes to the prop-types. 

I guess it is a matter of style (and bundle size) if we want to explicitly type out the prop types everywhere.